winner.rb

```diff
@@ -1,5 +1,4 @@
 require_relative 'tictactoe'
-
 class Win < Tictactoe
 	@@winx = Array.new 
```

Comment:
@Taniuchis Hello, I think you need instead of a class variables, an instance variable for example:

``` ruby
class Person
  # attr_reader :name # this line is like a getter
  # attr_writer :name # this line is like a setter
  attr_accessor :name # this line is both above

  # In ruby this is the constructor 
  def initialize(name, age)
    @name = name
    @age = age
  end

  def show_name
    name
  end

  def show_age
    age
  end
end

person = Person.new('Rogelio', 20)
puts person.show_name
person.name = 'Roy'
puts person.show_name
```

Try to use instance variables instead of class variables